pkg/controller/migmigration/task.go

```diff
@@ -164,9 +161,6 @@ var CancelItinerary = Itinerary{
 		{phase: DeleteRestores},
 		{phase: EnsureStagePodsDeleted, all: HasStagePods},
 		{phase: EnsureAnnotationsDeleted, any: HasPVs | HasISs},
```

Comment:
@aufi it looks like rollback is supposed to run `EnsureAnnotationsDeleted` already, so do you think we can remove this? Please check what runs in the phase to see if `EnsureAnnotationsDeleted` will fail during rollback due to having missing context (e.g. if we are using a MigMigration UID instead of a MigPlan UID to lookup resources we want to delete annotations on.

Reply:
Annotations are based on an object relation to Namespace where it belongs (no labels, even correlationLabels dependent on MigMigration/MigPlan's UUIDs, are used)